blog/2024/August/why-meshtastic-uses-managed-flood-router.mdx

```diff
@@ -0,0 +1,38 @@
+
+## Inefficiency is Limited
+A routing protocol aims to deliver packets at the destination in the most efficient way possible. Indeed for packets with a single destination, a lot can be gained by choosing only one efficient route. However, for broadcasts the potential gain is limited, as it needs to arrive at every node in the mesh. Since the majority of packets in Meshtastic are still broadcasts rather than messages targeting a specific node, a smarter protocol would have limited influence. Additionally, in a wireless medium, even if a packet is directed to a certain node, all nodes in range will witness it and during that time they cannot transmit or receive another packet, meaning that even with a _perfect_ routing protocol, nodes that may not be interested in a packet will still receive it, especially if they have good receiver sensitivity, which is frequently the case for LoRa.
+
+## Provides Scalability on low-bandwidth LoRa transport
+Furthermore, Managed Flood Routing is certainly scalable, because nodes that are unlikely to contribute to routing won’t participate. In large-scale mesh networks, where hundreds of devices are meshing, scalability becomes a critical factor and minimizing airtime is king. Traditional routing algorithms often struggle to handle the increasing number of devices and the associated routing overhead, due to the additional control messages required to maintain routes with constantly changing topology. On the other hand, with Managed Flood Routing, new devices joining the network integrate into the routing process without any additional ceremony, contributing to the overall network resilience and message delivery efficiency. 
+
+## Functions on Resource Constrained Devices
+Finally, Managed Flood Routing minimizes footprint on very resource-constrained low power IoT devices. Traditional routing algorithms often require devices to maintain routing tables or perform complex calculations, consuming additional RAM, flash, and valuable computational resources. Managed Flood Routing, with its simplicity and distributed nature, significantly reduces the resource overhead. Devices only need to forward messages they receive, without the need for complex computations, resulting in more device resource availability for other features and improved device autonomy.
+
+:::info Many decisions are based on empirical data from [Meshtasticator](https://github.com/GUVWAF/Meshtasticator) simulations
```

Comment:
Of the info block? Yes. Ideally, I wanted this in a blockquote style section, but it kept looking like arse because it was dropping quote characters around the entire contents